cpg_workflows/metamist.py

```diff
@@ -89,6 +89,29 @@
 _metamist: Optional['Metamist'] = None
 
 
+def silent_query(query_to_run, query_params: dict | None = None):
+    """
+    Run a query, but don't log the results to the console
+    Ref: https://github.com/populationgenomics/metamist/issues/540
+    Allow for a config override to log the results to the console
+
+    Args:
+        query_to_run (gql DocumentNode): Query String
+        query_params (dict): Query Parameters or None
+
+    Returns:
+        query result, with or without logging
+    """
+    if get_config()['workflow'].get('log_metamist', False):
```

Comment:
We should name this file better, but until we have a better solution, can all new parameters be added to `defaults.toml` with documentation (a comment) on what they do.

Reply:
Added the defaults.toml entry